custom/icds_reports/static/js/topojsons/README.md

```diff
@@ -3,9 +3,11 @@
 | File Name | Description | Used by | Format |
 | --------- | ----------- | ------- | ------ |
 | states_v2.topojson.js | Raw state shapes for the entire country | Master reference file only | JavaScript | 
-| states_v3_small.topojson.js | Optimized state shapes for the entire country | Web and mobile dashboard | topojson |
+| states_v3_small.topojson | (Deprecated) Optimized state shapes for the entire country | - | topojson |
```

Comment:
Planning to delete these files in a week once it's clear there aren't any issues.